convert2rhel/subscription.py

```diff
@@ -973,8 +973,10 @@ def check_needed_repos_availability(repo_ids_needed):
         if repo_id not in avail_repos:
             # TODO: List the packages that would be left untouched
             loggerinst.warning(
-                "%s repository is not available - some packages"
-                " may not be replaced and thus not supported." % repo_id
+                "Some repositories are not available: %s. Some packages may not be replaced with"
+                "their corresponding RHEL packages when converting. The converted system will be"
```

Comment:
```suggestion
                "their corresponding RHEL packages when converting. The converted system might be"
```

Reply:
Good catch on the phrasing I will get that changed, I will also work on the changes to the message in the google doc and change/push the new message!